DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: polars
 Title: Polars ported to R
-Version: 0.6.1.9000
+Version: 0.6.0.9000
```

Comment:
@etiennebacher I see, perhaps you have a point about here?
I really dont know how to undo this other than force push.